server/config.js

```diff
@@ -36,6 +36,11 @@ const conf = convict({
     format: ['production', 'development', 'test'],
     default: 'development',
     env: 'NODE_ENV'
+  },
+  max_file_size: {
+    format: Number,
+    default: (1024 * 1024 * 1024) * 2,
+    env: 'P2P_MAX_FILE_SIZE'
```

Comment:
but it's a new variable, is it not? unless you have an .env file that I am unaware of?